Library/Formula/crush-tools.rb

```diff
@@ -1,21 +1,23 @@
-require 'formula'
-
 class CrushTools < Formula
-  homepage 'http://crush-tools.googlecode.com/'
-  url 'https://crush-tools.googlecode.com/files/crush-tools-2013-04.tar.gz'
-  version '2013-04'
-  sha1 'a03a9d4719e8e049d836413598b636fd00f6a4cc'
+  homepage "https://github.com/google/crush-tools/"
+  url "https://github.com/google/crush-tools/archive/2013-04.tar.gz"
+  version "2013-04"
+  sha256 "55b80440b5892ec9ac9b812c72a615bc8455d4928540e75baf2513fb14ad358a"
 
-  depends_on 'pcre'
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pcre"
 
-  conflicts_with 'aggregate', :because => 'both install an `aggregate` binary'
+  conflicts_with "aggregate", :because => "both install an `aggregate` binary"
 
   def install
     # find Homebrew's libpcre
-    ENV.append 'LDFLAGS', "-L#{HOMEBREW_PREFIX}/lib"
+    ENV.append "LDFLAGS", "-L#{HOMEBREW_PREFIX}/lib"
 
+    system "./bootstrap"
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
   end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!